src/Invoice/InvRecurring/CronTokenRepository.php

```
@@ -0,0 +1,45 @@
+ * unchanged from the previous URL-query-parameter design — only the
+ * transport changes: `Authorization: Bearer <cron_key>` instead of
+ * `?cron_key=<cron_key>` in the URL, so the secret no longer sits in the
+ * URL where it leaks into shell history, cron logs, and web server access
+ * logs.
+ *
+ * @see CronIdentity
+ * @see \App\Invoice\InvRecurring\InvRecurringController::cron()
+ */
+final class CronTokenRepository implements IdentityWithTokenRepositoryInterface
+{
+    public function __construct(private readonly SR $sR) {}
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $sR. Configured minimum length is 3. (undefined)

(ShortVariable)